package.json

```diff
@@ -41,6 +41,7 @@
     ]
   },
   "devDependencies": {
+    "@reach/router": "^1.3.4",
```

Comment:
Are we not using `react-router` for routing purposes? I think that's the most used lib for routing.

Also, since routing is going to be the core part of the app, I think the package should be installed as the `dependencies` and not in the `devDependencies`.

Reply:
I'm not sure if we should have react-router at all. That feels like like an implementation detail a UI library wouldn't have.

Reply:
Ideally, we don't need routing in a UI library. I agree with that.

Reply:
It isn't `react-router` that there is in the `package.json` file. Or is it a typo?

Reply:
It's not a typo. He is using another package `reach-router` which also provides the solution for routing for React Apps.

Reply:
Oh!

Thanks for clarifying